nginx/server/vars/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 nginx_apt_repo: 'ppa:nginx/stable'
 
-nginx_apt_packages:
+nginx_package_names:
 - nginx
```

Comment:
I would suggest to move the package name into Debian and Redhat. That the names are the same, does not matter - they change independently.